includes/bfcache-opt-in.php

```diff
@@ -492,3 +498,34 @@ function filter_nocache_headers( $headers ): array {
 	__NAMESPACE__ . '\filter_nocache_headers',
 	1000
 );
+
+
+/**
+ * Conditionally adds the Permissions-Policy header to enable BFCache.
+ *
+ * This function checks if the 'bfcache_block_unload_events' option is enabled.
+ * If it is, the Permissions-Policy header with 'unload=()' is added to the HTTP
+ * response. This prevents 'unload' event listeners from firing, allowing the
+ * browser's Back/Forward Cache (BFCache) to function and improve navigation
+ * performance.
+ *
+ * @param array<string, string>|mixed $headers An array of HTTP headers to be sent.
+ * @return array<string, string> The filtered array of headers.
+ * @since n.e.x.t
```

Comment:
```suggestion
 * @since n.e.x.t
 * @access private
 *
 * @param array<string, string>|mixed $headers An array of HTTP headers to be sent.
 * @return array<string, string> The filtered array of headers.
```